source/includes/options-mongosqld.yaml

```
@@ -789,6 +789,20 @@ optional: true
 default: First IP address for :setting:`net.bindIp`.
 ---
 program: mongosqld
+name: gssapiConstrainedDelegation
+directive: option
+description: |
+  .. versionadded:: 2.11
+
+  Use proxy credentials for Kerberos authorization, enabling constrained
+  delegation. Requires :binary:`~bin.mongosqld` service credentials to be
+  present in the client keytab. See :ref:`configure-kerberos` for more
```

```suggestion
  present in the client keytab as well as the service keytab. See :ref:`configure-kerberos` for more
```
Just to make it clear that this is "in addition to" not "instead of"

---

Done, thanks Ryan!